src/tranquilo/history.py

```diff
@@ -173,6 +173,11 @@ def get_fvals(self, x_indices):
         )
         return out
 
+    def get_n_evals(self, x_indices):
+        fvals = self.get_fvals(x_indices)
+        n_evals = {k: len(v) for k, v in fvals.items()}
+        return n_evals
+
```

Comment:
I think in the future, we should write docstrings for these getter methods that explain what you're getting in what cases (scalar, least_squares, noisy, non-noisy,...).